opencog/persist/memcache/AtomCache.cc

```diff
@@ -30,7 +30,9 @@
 
 using namespace opencog;
 
-AtomCache::AtomCache(const std::string server, int portno)
+AtomCache::AtomCache(const std::string server, int portno) :
+    maxdepth(0),
+    load_count(0)
 {
```

Comment:
All of the code in the memcache directory is based on an ancient design and will not actually work any longer.  Its probably best to discard the entire directory.

Reply:
OK, less code --- less problems.